lambda.tf

```diff
@@ -0,0 +1,55 @@
+module "curator" {
+  source = "github.com/everydayhero/npm-lambda-packer"
+
+  package = "logar-curator"
+  version = "1.0.1"
+
+  environment = <<ENVIRONMENT
```

Comment:
Lambda environment can now be [specified](https://www.terraform.io/docs/providers/aws/r/lambda_function.html#environment) in the resource. Useful in future